### PR TITLE
add Intel Icelake

### PR DESCRIPTION
**Pull request by mdavidsaver**
Attempt to address #2610

### CODE REVIEW

File: src/PerfCounters.cc [hunk 1]
```diff
@@ -121,6 +122,7 @@ struct PmuConfig {
 // See Intel 64 and IA32 Architectures Performance Monitoring Events.
 // See check_events from libpfm4.
 static const PmuConfig pmu_configs[] = {
+  { IntelIcelake, "Intel Icelake", 0x5111c4, 0, 0x5301cb, 0, 100, PMU_TICKS_RCB },
```
Comment (khuey) [inline src/PerfCounters.cc:125]: Can we replace 0x5301cb with 0 here? I don't think that counter actually exists on Ice Lake.

Comment (mdavidsaver) [reply src/PerfCounters.cc]: Done.  Zero seems to make no difference.

Comment (khuey) [reply src/PerfCounters.cc]: Yeah that entry in the table isn't actually used now.

**Comment by khuey** [comment]
Thanks for the PR and testing!

**Comment by mdavidsaver** [comment]
Thanks for taking the time with this.

**Pull request merged**